src/components/common/ForeignJurisdiction.vue

```diff
@@ -40,6 +42,12 @@ import { CountriesProvincesMixin } from '@/mixins'
 
 @Component({})
 export default class ForeignJurisdiction extends Mixins(CountriesProvincesMixin) {
+    // Refs
+    $refs!: {
+      countrySelectRef: any
+      regionSelectRef: any
+  }
+
```

Comment:
add refs for validations

Reply:
OK, these refs are now used in THIS component for validation: 👍